R/ipcw.R

```diff
@@ -0,0 +1,34 @@
+# ------------------------------------------------------------------------------
+# Functions for using inverse probability of censoring weights (IPCW) in
+# censored regression models
+
+# ------------------------------------------------------------------------------
+# Simple helpers for computing the probability of censoring
+
+# For avoiding extremely large, outlier weights
+trunc_probs <- function(probs, trunc = 0.01) {
+  complt_prob <- !is.na(probs)
+  probs_non_zero <- probs[complt_prob]
+  non_zero_min <- min(probs_non_zero[probs_non_zero > 0])
```

Comment:
I was mildly confused by `probs_non_zero` because those are not actually non-zero but non-missing. Wouldn't this do what you want?
```suggestion
  non_zero_min <- min(probs[probs > 0], na.rm = TRUE)
```

Reply:
I can rename to make it more clear but I think the issue is that, if there are missing values, `probs > 0` gives us missing values.